0.18/files/create-settings.sh

```diff
@@ -249,17 +249,17 @@ if [[ ${FORCE_GENERATE_SETTINGS_FILES:-} ]]
 then
 
 	#backup old files using timestamp.
-	if [[ ! -f $CONFIG/server-settings.json ]]
+	if [[ -f $CONFIG/server-settings.json ]]
 	then
 		mv "$CONFIG/server-settings.json" "$CONFIG/server-settings.json.$(date +%Y.%m.%d.%H.%M.%S)"
 	fi
 
-	if [[ ! -f $CONFIG/map-gen-settings.json ]]
+	if [[ -f $CONFIG/map-gen-settings.json ]]
 	then
 		mv "$CONFIG/map-gen-settings.json" "$CONFIG/map-gen-settings.json.$(date +%Y.%m.%d.%H.%M.%S)"
 	fi
 
-	if [[ ! -f $CONFIG/map-settings.json ]]
+	if [[ -f $CONFIG/map-settings.json ]]
```

Comment:
I think those changes are a mistake.

Reply:
Basing off of https://linuxize.com/post/bash-check-if-file-exists/ and that i was getting errors when I had it the other way.

Reply:
I don't know why it should error but right now it checks if the file does exist and then copies the default file but we want to do it when the file does not exist.

Reply:
This is the section that backs up an _existing_ settings file (if you force the update of the settings via FORCE_GENERATE_SETTINGS_FILES). As such, if the file exists, we want to back it up. Lines 278-295 are the ones that you might be confusing these with. those are the ones that only move the generated settings file in if a file does not already exist.